README.rst

```diff
@@ -8,141 +8,225 @@ PHP PDO driver for Snowflake
 .. image:: http://img.shields.io/:license-Apache%202-brightgreen.svg
     :target: http://www.apache.org/licenses/LICENSE-2.0.txt
 
-*Private Preview. Linux Only. No PHP 5 support. PHP 7.2 only.*
+Snowflake provides a driver that uses the 
+`PHP Data Objects (PDO) extension <https://www.php.net/manual/en/book.pdo.php>`_.
+to connect to the Snowflake database.
 
-Configuring Environment
+.. contents::
+
+Prerequisites
 ================================================================================
 
-PHP Versions and Extensions
-----------------------------------------------------------------------
+In order to build the Snowflake PHP PDO Driver, you must have the following software installed:
 
-PHP 7.2 is supported. The following extensions are required:
+- (Windows) Visual Studio
+- 
 
-    * pdo
-    * json
-    
-If you're not building PHP from source, you will also need to install the corresponding PHP development package for your OS/environment
+In order to install and use the Snowflake PHP PDO Driver, you must have the following software installed:
 
-Application Server (Optional)
-----------------------------------------------------------------------
+- PHP 7.2
+- the :code:`php-pdo` extension
+- the :code:`php-json` extension
 
-If the PHP is used along with an application server, e.g., nginx, apache, which is the common use case, install them and configure the PHP handler accordingly so that the PHP file can run on the web.
-For example, Nginx enables the PHP handler by adding the following config in :code:`/etc/nginx/sites-available/default`:
+.. note::
 
-.. code-block:: text
+  The rest of the instructions assume that you have also installed the ``php-fpm`` extension.
```

Comment:
We should also state here that it works with the regular PHP CGI and that you don't need `php-fpm`

Reply:
Thanks! I added this.